src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.props

```diff
@@ -10,6 +10,6 @@ Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\analyzers\build\Microsoft.CodeAnalysis.NetAnalyzers.props"
+  <Import Project="$(MSBuildThisFileDirectory)..\analyzers\build\Microsoft.CodeAnalysis.NetAnalyzers.props"
```

Comment:
Project imports should already be relative to the current file.  Was something failing without this change?

Reply:
it was not. I verified that this does the right though without this change using -pp. I'll back this out. I was being paranoid.